src/static/js/cart.js

```diff
@@ -0,0 +1,78 @@
+const { ipcRenderer } = require('electron');
+
+let cartList = new Array();
+let cost = 0;
+
+// Puts the given product in the cart or increments it if
+// it's already present.
+module.exports.AddToCart = (product) => {
+  let alreadyInCart = cartList.filter(prod => prod.id === product.id);
```

Comment:
Use some instead of filter this is a over complicated boolean

Reply:
Yea agree, fixed it